src/services/geetest.types.d.ts

```diff
@@ -1,3 +1,5 @@
+import { GeetestError } from "@domain/geetest/error"
```

Comment:
Imports should be done like this so that we don't lose the implicit imports from this file.

We should also probably case this as `GeeTest..` to be consistent with other types.

```suggestion
type GeeTestError = import("@domain/geetest/error").GeeTestError
```

Reply:
> Imports should be done like this so that we don't lose the implicit imports from this file.

I don't really understand this. where can I learn more about it?

Reply:
I think I get some of it. if it's in a `d.ts` then type is being automatically added, but only if this is a `type` not an `import`?